src/black/comments.py

```diff
@@ -271,7 +276,7 @@ def contains_pragma_comment(comment_list: List[Leaf]) -> bool:
         pylint).
     """
     for comment in comment_list:
-        if comment.value.startswith(("# type:", "# noqa", "# pylint:")):
+        if re.match(r"#\s*(type\s*:|pylint\s*:|noqa\s*)", comment.value):
```

Comment:
```suggestion
        if re.match(r"#\s*(type\s*:|pylint\s*:|noqa\s*|nosec\s*)", comment.value):
```
Would be nice if [Bandit](https://bandit.readthedocs.io/en/latest/config.html#suppressing-individual-lines) is supported as well.